tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -1063,6 +1064,12 @@ private function getOrderDiscounts(int $orderId): array
      */
     private function getOrderDiscountByName(int $orderId, string $cartRuleName): ?OrderDiscountForViewing
     {
+        $cartRuleName = CommonFeatureContext::getContainer()->get('translator')->trans(
```

Comment:
Are you sure of this one? Since OrderCartRule only have one name (lot localized), I'm afraid it won't match any more with the english name in database?

Reply:
In https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Cart/CommandHandler/SetFreeShippingToCartHandler.php#L118 the cart rule name is translated. That's why I had 'Livraison gratuite' and not 'Free shipping'

Reply:
Oh ok, what worries me is that it might be the only one 😅 
Besides the default language should be english, but nevermind, as long as the test suite is happy we can always fix it later if necessary